oak_idl_tests/tests/generate.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#[test]
+fn generate() {
+    // This test simply copies the output created by the `build.rs` script back into the source
+    // tree, so that it can be tracked in Git. We use the `.txt` extension so that other tools do
```

Comment:
Related to my comment above: why do we not want this code to be checked? 

I think we want this code to be checked.

Reply:
See my other comment. We definitely don't want this to be checked, it's just a string from the point of view of the generator. E.g. we will add C++ generation at some point, and we don't want to apply our arbitrary formatting rules to it, since it will be used in other contexts (and not read by humans, in general). It's like generated protobuf code, or any other intermediate representation.